docs/software-platforms/aws/index.md

```diff
@@ -3,24 +3,31 @@
 
 !!! note
-    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about AWS access.
+    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about getting AWS access.
 
-## Joining IAM Identity Center
+    Once we have created an AWS account for you, you can proceed to [set up your account](./joining-aws.md).
+    Note that your use of AWS services described in this section counts towards your [monthly budget](../../getting-started/accessing-resources/getting-access-to-compute.md#monthly-budget).
 
-When we create an account for you, we will also create a user for you with the username `researcher-{your github handle}`.
+## S3: Data storage with AWS
 
-You will receive an email from `no-reply@login.awsapps.com` with the subject "Invitation to join AWS IAM Identity Center (successor to AWS Single Sign-On)."
+We use [AWS S3](https://aws.amazon.com/s3/) to store project data in folders referred to in S3 as "buckets."
+We also use S3 as central location for you to share your result files with us before [filing a pull request](../../contributing-to-analyses/creating-pull-requests/index.md).
+All contributors will have a designated [researcher bucket](working-with-s3-buckets.md) where results can be synced and shared as part of [code review](../../contributing-to-analyses/pr-review-and-merge/index.md).
 
-You must accept this invitation to gain access to your AWS account and to download project data from S3.
+Please refer to these pages about working with S3:
 
-To accept the invitation, please take the following steps.
+- [Locally logging into AWS](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session)
+- [Accessing project data from S3](../../getting-started/accessing-resources/getting-access-to-data.md#accessing-data-on-s3)
+- [Using your researcher bucket](working-with-s3-buckets.md)
 
-1. Note the following information in the invitation email: **Your AWS access portal URL**.
-   You will need this value to [configure AWS CLI locally](../../technical-setup/environment-setup/configure-aws-cli.md) and to [log into the console to use Lightsail for Research](STUB_LINK).
-   It is also helpful to bookmark this URL so you can easily return to it.
+## LSfR: Virtual computing with AWS
 
-2. Click the `Accept Invitation` button in the email.
+As an OpenScPCA contributor, you are eligible for a monthly allocation on [Lightsail for Research (LSfR)](https://aws.amazon.com/lightsail/research/), an Amazon product that allows you to work on a virtual Ubuntu (Linux) Desktop computer in your browser.
 
-3. Enter a strong password that is unique to this service when prompted in the browser.
 
-4. Set up a multifactor authentication device, such as an authenticator app or security key, when prompted in the browser.
+Please refer to these pages about using LSfR:
```

Comment:
```suggestion
Please refer to these pages to learn more about setting up LSfR:
```
Since they need to follow these links to get set up (or at least the first three) what do we think about rephrasing this?

Reply:
how about `...to learn more about setting up and using LSfR`?

Reply:
https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/344/commits/c9a0231c7785dc4f415f2ebfe52febaf8921f025

Reply:
I like this!